app/api.js

```diff
@@ -40,6 +39,7 @@ SELECT ?period ?periodLabel ?number WHERE {
   ?period p:P31 ?thing .
   ?thing pq:P642 wd:Q571436.
   ?thing pq:P1545 ?number .
+  FILTER(xsd:integer(?number) >= 14 && xsd:integer(?number) <= 15).
```

Comment:
Maybe you can add a short comment above the function that explains why we filter for this range here? That might remind us to remove the filter at some point when the data has been added for the other terms as well.
After that, I'm super happy to merge this!

Reply:
Just in case you missed it, I updated my commit with your suggestion.

Reply:
Thanks for pointing this out. It seems like Github does not send notification emails for force pushes unfortunately.